extensions/vertx-http/deployment/src/main/java/io/quarkus/vertx/http/deployment/HttpRootPathBuildItem.java

```diff
@@ -0,0 +1,26 @@
+package io.quarkus.vertx.http.deployment;
+
+import io.quarkus.builder.item.SimpleBuildItem;
+
+public final class HttpRootPathBuildItem extends SimpleBuildItem {
+
+    private final String rootPath;
+
+    public HttpRootPathBuildItem(String rootPath) {
+        this.rootPath = rootPath;
+    }
+
+    public String getRootPath() {
+        return rootPath;
+    }
+
+    public String adjustPath(String path) {
+        if (!path.startsWith("/")) {
```

Comment:
What's the point here? Can't we add the initial `/` if missing? It's pretty obvious the root path must have a `/` because that's just how URIs are built, so it's not like this could be mistaken for a slashless root path…